### PR TITLE
Filecmd => 5.39

### PR DESCRIPTION
**Pull request by supechicken666**
Tested on `x86_64`

### CODE REVIEW

File: packages/filecmd.rb [hunk 1]
```diff
@@ -38,7 +29,12 @@ def self.check
   end
   
   def self.install
-    system "install -Dm755 filefix #{CREW_DEST_PREFIX}/bin/filefix"
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install-strip"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.install 'filefix', "#{CREW_DEST_PREFIX}/bin/filefix", mode: 0755
+    system "make DESTDIR=#{CREW_DEST_DIR} install-strip | sed -e \"s,libtool: warning: remember to run 'libtool --finish #{CREW_LIB_PREFIX}',,g\""
```
Comment (uberhacker) [inline packages/filecmd.rb:34]: Why do we need to remove the warning?

Comment (supechicken666) [reply packages/filecmd.rb:34]: We already done it in `postinstall` part

**Comment by uberhacker** [comment]
This is a core package so we will need new binaries.

**Comment by satmandu** [comment]
Here are binaries uploaded for you:
```
  binary_url ({
     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/filecmd-5.39-chromeos-armv7l.tar.xz',
      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/filecmd-5.39-chromeos-armv7l.tar.xz',
        i686: 'https://dl.bintray.com/chromebrew/chromebrew/filecmd-5.39-chromeos-i686.tar.xz',
      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/filecmd-5.39-chromeos-x86_64.tar.xz',
  })
  binary_sha256 ({
     aarch64: 'e87bec0e97e45cdf47a1badf36157c4c11565b2a4da081c653d79a26fb4099f6',
      armv7l: 'e87bec0e97e45cdf47a1badf36157c4c11565b2a4da081c653d79a26fb4099f6',
        i686: '85396091dd796294e2b586036153df826bfa5e7007327779fa8972c828cdc445',
      x86_64: 'a27d190d2032ce8c378b7ee902e9532b6be55498bd4b6e7bb48e65d492239159',
  })
```

File: packages/filecmd.rb [hunk 2]
```diff
@@ -38,7 +42,12 @@ def self.check
...
+  end
+  
+  def self.postinstall
+    system "libtool --finish #{CREW_LIB_PREFIX}"
```
Comment (uberhacker) [inline packages/filecmd.rb:51]: Does it make sense to add this in `crew`?

File: packages/filecmd.rb (hunk 1 above)
Comment (uberhacker) [reply packages/filecmd.rb:47]: I think it would be better if we kept the warning message.  What does it hurt?

File: packages/filecmd.rb (hunk 2 above)
Comment (satmandu) [reply packages/filecmd.rb:51]: I've seen lots of packages suggest that...

Comment (uberhacker) [reply packages/filecmd.rb:51]: I mean incorporate this logic in the `crew` command.

**Comment by uberhacker** [comment]
Fixed by #4991.

**Pull request closed without merging**